keyboards/keebio/viterbi/keymaps/ericgebhart/rules.mk

```diff
@@ -0,0 +1 @@
+DEFAULT_FOLDER = keebio/viterbi/rev1
```

Comment:
File should be deleted

Reply:
Can you explain why I cannot set my own default folder?

Reply:
Its use is for backwards compatibility, and to cover short term issues in things like CI, not the use case you have. The fact it works is more luck than by design.

Reply:
ah. well. then it's moot.  My keyboard seems to be working regardless.
It just seemed like the right thing.